benchmark_runner/common/oc/oc.py

```diff
@@ -87,13 +87,24 @@ def get_free_disk_id(self):
         if workers_disk_ids:
             return list(set(workers_disk_ids) - set(workers_pv_disk_ids))
 
-    def get_kata_version(self):
+    def get_kata_operator_version(self):
         """
-        This method returns kata version
+        This method returns kata operator version
         :return:
         """
         return self.run(f"{self.__cli} get csv -n openshift-sandboxed-containers-operator $({self.__cli} get csv -n openshift-sandboxed-containers-operator --no-headers | awk '{{ print $1; }}') -ojsonpath='{{.spec.version}}'")
 
+    @typechecked
+    def get_kata_rpm_version(self, node: str):
+        """
+        This method returns sandboxed containers (kata) rpm version
+        @param: node
+        :return:
+        """
+        kata_rpm_version = self.run(f"{self.__cli} debug node/{node} -- chroot /host rpm -q kata-containers 2>/dev/null")
+        kata_version_list = kata_rpm_version.split('-')[2:4]
+        return f"{kata_version_list[0]}-{kata_version_list[1].split('.')[0]}"
```

Comment:
You can do this, and avoid any need to parse the output:

```
return self.run(f"{self.__cli} debug node/{node} -- chroot /host rpm -q --queryformat='%{VERSION}-%{RELEASE}.%{ARCH}' kata-containers 2>/dev/null")
```

Reply:
@RobertKrawitz please review my fix